database/guests.ts

```diff
@@ -1,17 +1,4 @@
-declare module globalThis {
-  let guestsDatabase: Guest[];
-}
-
-// Initialize database to empty array only once
-//
-// TODO: Add a full PostgreSQL database connection
-if (!('guestsDatabase' in globalThis)) {
-  globalThis.guestsDatabase = [];
-}
-
-const guests = globalThis.guestsDatabase;
-
-console.log('guests in db', guests);
+import { sql } from './connect';
 
 export type Guest = {
```

Comment:
this should be in the migration file

Reply:
Moved it to the migration file
[832a0bb](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/832a0bbbd2a8f312f8bcf35dd3f2e868a49317dd)